rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -907,3 +907,56 @@ def check_sync_status(retry=None, delay=None):
     is_multisite = utils.is_cluster_multisite()
     if is_multisite:
         sync_status()
+
+
+def get_default_datalog_type():
+    """
+    get the default datalog type i.e. omap or fifo
+    """
+    cmd = "ceph config get mon.* rgw_default_data_log_backing"
+    default_datalog_type = utils.exec_shell_cmd(cmd)
+    if default_datalog_type is False:
+        raise TestExecError("Error in getting the default datalog backing type")
```

Comment:
i think the exception should be different here. not `TestExecError`. maybe create a new one.

Reply:
raised a new exception DefaultDatalogBackingError